PackageViewModel/PackageViewModel.cs

```diff
@@ -601,9 +622,40 @@ private void DeleteContentExecute(object parameter)
         {
             DiagnosticsClient.TrackEvent("PackageViewModel_DeleteContentExecute");
 
-            if ((parameter ?? SelectedItem) is PackagePart file)
+            // Null objects are put sometimes into List, so to not count them
```

Comment:
Do you know how to reproduce this?

Reply:
As I wrote down into commit description. This occur only when user creates new file / folder into existing folder. If I create new file and not putting it right into some folder, it does not happen. I tried to check it out, or remove those ones on creating of new file, but it did not work.